test/authentication.js

```diff
@@ -0,0 +1,41 @@
+let chai = require('chai');
+let chaiHttp = require('chai-http');
+const expect = require('chai').expect;
+
+// DOCUMENTATION
+
+// Test for authentication is created
+// The first test brings the result OK with the user data
+// The second test brings an error for not having TOKEN
+
+chai.use(chaiHttp);
+
+const url = 'http://localhost:3000';
+let token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJmaXJzdE5hbWUiOiJsdWNhcyIsImxhc3ROYW1lIjoiUml2YWRlbmVpcmEiLCJlbWFpbCI6ImxybGljaGFyZGlAaG90bWFpbC5jb20iLCJwYXNzd29yZCI6IiQyYiQxMCQ5cEsuekdLcjFNMGJKNXo5U3hxcVJla0NDSVIxdHBLVmlkQU9FQnJMRjFLd0NoSlZnalJzMiIsInJvbGVJZCI6MiwiaWF0IjoxNjQ4NzY3MzA5LCJleHAiOjE2NDg4NTM3MDl9.EOXM4vxoRUGVgtMHV2fzIVMWHOkw3NKxeH9GOotYk3c";
+
+ describe('Get data user specific', () => {
+    it('Should respond with json containing data user specific', (done) => {
+      chai.request(url)
+      .get('/auth/me')
+      .set({'Authorization': token})
+      .end((err , res) => { 
+        console.log(res.body)
+        expect(res).to.have.status(200);
+        done()
+      })
+    });
+  });
+
+  describe('Error at get data user specific for token', () => {
```

Comment:
no hace falta crear un describe. La idea delos describes es poder dividir los tests en tipos de test. La descripción de ambos describes son los it, o sea, lo que deberia pasar. Lo que podés hacer es.. crear un describe para probar varias respuestas con status 200, y luego otro describe para probar respuestas con status de error.. entonces lo describes podrian ser:
```js
describe('Ok responses', () => ...);

describe('Error responses', () => ...);
```

Y dentro de cada uno, los it correspondientes a cada prueba para probar distintas cosas.

Es un ejemplo lo que digo

Reply:
perfecto si si es la unica prueba que le hago una de ok response y otra de error por falta de token